### PR TITLE
Add Side-to-Side Navigation content

### LINKED ISSUE #2089: Navigation -Create Side-by-Side section

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-2209--keen-mayer-a86c8b.netlify.app/templates/side-to-side-navigation)
[Figma](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9562)

#### What does this PR do?
Adds content and examples for Side-to-Side Navigation. I am trying to figure out how to add in the "sidebar" visual in a way that doesn't eat up so much screen real estate. Since it is such a narrow and tall image, it creates a lot of dead space to the right of it.

#### Where should the reviewer start?
aries-site/src/pages/templates/side-to-side-navigation.mdx

#### What testing has been done on this PR?
Local deploy.

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?
Check deploy.

#### What are the relevant issues?
Closes #2089 

#### Should this PR be mentioned in Design System updates?
Yes.

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.

### CODE REVIEW

File: aries-site/src/pages/templates/side-to-side-navigation.mdx [hunk 1]
```diff
@@ -1,3 +1,89 @@
-import { ComingSoon } from '../../components';
+import { Box, Image, NameValueList, NameValuePair } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
 
-<ComingSoon />
\ No newline at end of file
+## When to use
+
+- Tabs to maneuver between relevant information within the same view
```
Comment (britt6612) [inline aries-site/src/pages/templates/side-to-side-navigation.mdx:6]: question: should we be putting periods? I added them to my page but to be consistent I wanted to ask

Comment (taysea) [reply aries-site/src/pages/templates/side-to-side-navigation.mdx:6]: yes we should, thank you!

**Comment by britt6612** [comment]
This image creates a horizontal scroll at a certain width
https://user-images.githubusercontent.com/42451602/149185499-da72aa3a-96ce-44e0-a8a0-9fed583705f8.mov

File: aries-site/src/pages/templates/side-to-side-navigation.mdx [hunk 2]
```diff
@@ -1,3 +1,101 @@
...
+- Global Header should be on all applications and websites
+- Allows users to move between top-level important destinations
+- Located at the top of all application and websites
+
+<Example 
+  previewWidth="xlarge"
+  height="xsmall"
+  plain
+>
+  <DiagramPreview
```
Comment (vavalos5) [inline aries-site/src/pages/templates/side-to-side-navigation.mdx:35]: I realize that the screenshot I have on Figma is probably not the best one as I didn't crop it and you can see the background color underneath. It also looks a little pixelated. I'll make a note of this for myself and replace the one in Figma. Once I've replaced it, i'll ping you.

Comment (taysea) [reply aries-site/src/pages/templates/side-to-side-navigation.mdx:35]: I actually pulled this from what we already have on the GlobalHeader page -- think the original screenshot came from a Windows screen which may be reason for seeming a little pixelated.

If you get an updated version, let me know and I'll replace.

Comment (vavalos5) [reply aries-site/src/pages/templates/side-to-side-navigation.mdx:35]: Got it! What i'll do is recreate it in Figma as a vector that way it performs better in different resolutions.

File: aries-site/src/pages/templates/side-to-side-navigation.mdx [hunk 3]
```diff
@@ -1,3 +1,101 @@
...
+- Hamburger menus, sidebars, nav bars, wizards.
+- Switching from one service to another service.
+- Sidebar to navigate between pages.
+- Accordions to open a single panel to navigating between content within the same page.
+
+## HPE Local Header Example
+
+The HPE product’s local header which contains grouped, frequently used and top-level information.
+
+Example coming soon.
```
Comment (britt6612) [inline aries-site/src/pages/templates/side-to-side-navigation.mdx:16]: wonder if we should make this bold to let the user know by glancing looks like content

**Comment by britt6612** [comment]
I would also add `Tabs` in related content

File: aries-site/src/pages/templates/side-to-side-navigation.mdx (hunk 3 above)
Comment (taysea) [reply aries-site/src/pages/templates/side-to-side-navigation.mdx]: Good call, bolded.

**Review by britt6612 — APPROVED** [review]
Looks good thanks for addressing changes

**Review by vavalos5 — APPROVED** [review]
LGTM!

**Review by halocline — APPROVED** [review]
No code related feedback.

**Pull request merged**